app/src/main/java/com/tuf2000m/energymeter/views/home/recenthistory/RecentHistoryAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.tuf2000m.energymeter.views.home.recenthistory
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.tuf2000m.energymeter.data.model.recent.RecentData
+import com.tuf2000m.energymeter.databinding.ItemRecyclerRecentsBinding
+
+interface OnItemClickListener {
```

Comment:
This interface defines a click listener for items in the RecyclerView. Implement it to handle item clicks.